### PR TITLE
chore(docs): add migration guide for upgrade to 3.9.0-preview-1

### PR DESCRIPTION
**Pull request by RomanJRW**

### CODE REVIEW

**Review by grlimacan — DISMISSED** [review]
Content-wise this looks good (except for one minor thing), but the pipeline is failing because of some other docs it seems. Pls get that sorted out before merging

File: versioned_docs/version-1.3/self-managed/optimize-deployment/migration-update/instructions.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ Roughly every quarter of a year a new minor version of Optimize is released. The
...
 
-| Update from | Recommended update path to 3.7 |
-| --- | --- |
-| 3.7 | You are on the latest version. |
-| 3.0 - 3.6 | 1. Rolling update to 3.6  <br /> |
-| 2.0 - 2.7 | 1. Rolling update to 2.7 <br /> 2. Rolling update from 2.7 to 3.0 |
-| 1.0 - 1.5 | No update possible. Use the latest version directly. |
+| Update from     | Recommended update path to 3.9.0-preview-1                        |
+| --------------- | ----------------------------------------------------------------- |
+| 3.9.0-preview-1 | You are on the latest version.                                    |
```
[the remark this answers is not in the document]
Comment (RomanJRW) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/migration-update/instructions.md:15]: @christinaausley / @akeller - does this file actually need updating?

[the remark this answers is not in the document]
Comment (RomanJRW) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/migration-update/instructions.md]: ok, removed those changes from this PR 👍 

@akeller - is it ok to proceed with merging despite the broken link failures? They are unrelated to any changes here

**Comment by akeller** [comment]
I'll merge main into your branch to fix the build issues and then get this merged. Thanks!

**Review by christinaausley — APPROVED** [review]
Added punctuation to one line but otherwise looks good 👍

**Pull request merged**